v1/translate.yaml

```diff
@@ -0,0 +1,254 @@
+info:
+  version: '1.0.0'
+  title: Content Translation API
+  description: Content Translation API
+  termsofservice: https://github.com/wikimedia/restbase
+  license:
+    name: Apache licence, v2
+    url: https://www.apache.org/licenses/LICENSE-2.0
+paths:
+  /machine/{from}/{to}{/provider}:
```

Comment:
It is not the kind of lingo people use in this field. To me it appears as if it is about some machinery related to translation (compare with _translation engine_), especially if it appears out of context like it does here. Machine translation is a thing, but if you swap the order of words it no longer makes sense. And like said above, there are no other forms of translations this would contrast with. Hence I think some repetition is the smallest evil.